config-connector/solutions/apps/helm/wordpress/README.md

```diff
@@ -0,0 +1,55 @@
+# Wordpress
+
+==================================================
+
+## NAME
+
+  wordpress
+
+## SYNOPSIS
+
+The WordPress application demonstrates how you can configure a WordPress site powered by GCP MySQL database and using Workload Identity for authentication.
+
+## CONSUMPTION
+
+  ```bash
+   helm install ./charts/wordpress-gcp/ --set google.projectId=[PROJECT_ID]
+  ```
+
+## REQUIREMENTS
+
+1. GKE Cluster with Config Connector and [Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#enable_workload_identity_on_a_new_cluster).
+1. [Helm](https://helm.sh/docs/using_helm/)
+
+## USAGE
+
+All steps are run from this directory.
+
+1. Review and update the values in `./charts/wordpress-gcp/values.yaml` .
+1. Validate and install the sample with Helm
+
+    ```bash
+    # validate your chart
+    helm lint ./charts/wordpress-gcp/ --set google.projectId=[PROJECT_ID]
+
+    # check the output of your chart
+    helm template ./charts/wordpress-gcp/ --set google.projectId=[PROJECT_ID]
+
+    # install your chart
+    helm install ./charts/wordpress-gcp/ --set google.projectId=[PROJECT_ID] --generate-name
```

Comment:
When I ran the command with latest Helm, it returned the following error:`Error: unknown flag: --generate-name` Is it a valid flag that we want to use?

Reply:
can you please run `helm version`, what does it show? Mine is `version.BuildInfo{Version:"v3.1.2", GitCommit:"d878d4d45863e42fd5cff6743294a11d28a9abce", GitTreeState:"clean", GoVersion:"go1.13.8"}`

Reply:
Updated root readme, mentioned Helm 3.0.